doc/dev/autograph.rst

```diff
@@ -1024,4 +1024,4 @@ The list of supported operators includes:
 - ``-=`` (add with negation)
 - ``*=`` (multiply)
 - ``/=`` (divide)
-- ``**=`` (power)
\ No newline at end of file
+- ``**=`` (power)
```

Comment:
What happened on this line? :sweat_smile:

Reply:
There was a missing newline at the end of the file :wink: